src/log_prob_util.jl

```diff
@@ -0,0 +1,490 @@
+###################################################################
```

Comment:
Once you move the functions from ```log_prob.jl``` to live with their VB counterparts, there won't be much left, and you can just combine what's left with these functions in a single ```log_prob.jl``` file.  This contains more than utilities, IMO -- it's much of the log likelihood calculation and types.